libs/ngxtension/computed-from/src/computed-from.spec.ts

```diff
@@ -58,6 +59,30 @@ describe(computedFrom.name, () => {
 				expect(s()).toEqual([1]);
 			});
 		});
+		it(`MD FIX for Observables that don't emit synchronously, computedFrom will THROW ERROR`, () => {
+			TestBed.runInInjectionContext(() => {
+				const late = of(1).pipe(delay(1000)); // late emit after 1s
+				expect(() => {
+					const s = computedFrom([late]);
+				}).toThrowError(/requireSync/i); //THROW ERROR NG0601 DUE TO toSignal + requireSync: true
+				//THIS WILL PREVENT OLD "SPURIOUS SYNC EMIT" OF null OR Input ([], {}) THAT CAN CAUSE TS RUNTIME ERRORS
```

Comment:
Can you use normal lowercase sentences, please? 

Uppercase is not easy to read.